app/commands/document/create_search_index_document.rb

```diff
@@ -0,0 +1,20 @@
+class Document::CreateSearchIndexDocument
+  include Mandate
+
+  initialize_with :doc
+
+  def call
+    {
+      id: doc.id,
+      title: doc.title,
+      blurb: doc.blurb,
+      markdown: doc.markdown,
+      updated_at: doc.updated_at,
```

Comment:
I originally used this for sorting, but seeing how people are always search for text I think, we could remove this. Unless you want to order by the updated at date if the score of documents is equal.

Reply:
I think sorting should be based on the ES match score, not on anything else.

Reply:
Perfect. I've removed the field.